http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoPubsubMessageSubscriber.scala

```diff
@@ -537,21 +537,17 @@ class LeoPubsubMessageSubscriber[F[_]: Timer: ContextShift: Parallel](
   ): F[Unit] =
     for {
       ctx <- ev.ask
+      _ <- logger.info(s"Beginning disk deletion for ${diskId} | trace id: ${ctx.traceId}")
       diskOpt <- persistentDiskQuery.getById(diskId).transaction
       disk <- diskOpt.fold(
         F.raiseError[PersistentDisk](PubsubHandleMessageError.DiskNotFound(diskId))
       )(F.pure)
-      _ <- if (disk.status != DiskStatus.Deleting)
-        F.raiseError[Unit](
-          PubsubHandleMessageError.DiskInvalidState(diskId, disk.projectNameString, disk)
-        )
-      else F.unit
       operation <- googleDiskService.deleteDisk(disk.googleProject, disk.zone, disk.name)
       whenDone = persistentDiskQuery.delete(diskId, ctx.now).transaction[F].void >> authProvider.notifyResourceDeleted(
         disk.samResource,
         disk.auditInfo.creator,
         disk.googleProject
-      )
+      ) >> logger.info(f"Completed disk deletion for ${diskId} | trace id: ${ctx.traceId}")
```

Comment:
Maybe want `s".."` -- I think the `f` interpolater is for `printf`-like syntax

Reply:
Oh, I was playing with python recently :)